src/test/java/se/sundsvall/datawarehousereader/service/CustomerServiceTest.java

```diff
@@ -340,12 +286,12 @@ void getDetailsWithCustomerEngagementOrgIdSet() {
 				true));
 
 		assertThat(result.getMetaData().getCount()).isEqualTo(1L);
-		assertThat(result.getMetaData().getLimit()).isEqualTo(100);
+		assertThat(result.getMetaData().getLimit()).isEqualTo(1);
```

Comment:
Why is this not 100?

Reply:
Seems to be a general fault in the test class as a whole, I'll look into it